_providers/hermes.radio.md

```diff
@@ -0,0 +1,14 @@
+---
+name: hermes.radio
+status: PREPARATION
```

Comment:
status `PREPARATION` requires a `before_login_hint` that describes the user what to do to make things working.

if there is nothing to do, just use the status `OK`.

(this has nothing to do with this file or the server is in "preparation", it is about preparations the user has to do on real servers (enable imap for gmx etc.)

Reply:
I'll change from preparation to OK.
Server info is not there yet, this is intended.

Reply:
Updated file with OK in status.